sql/timeseries/pwaScores.sql

```diff
@@ -0,0 +1,44 @@
+#standardSQL
+CREATE TEMPORARY FUNCTION
+  getPWAScore(report STRING)
+  RETURNS FLOAT64
+  LANGUAGE js AS """
+$=JSON.parse(report);
+return $.reportCategories.find(i => i.name === 'Progressive Web App').score;
+""";
+SELECT
+  date,
+  UNIX_MILLIS(TIMESTAMP(REGEXP_REPLACE(date, "_", "-"))) AS timestamp,
+  client,
+  ROUND(APPROX_QUANTILES(lighthouse_pwa_score, 101)[
+  OFFSET
+    (11)], 2) AS p10,
+  ROUND(APPROX_QUANTILES(lighthouse_pwa_score, 101)[
+  OFFSET
+    (26)], 2) AS p25,
+  ROUND(APPROX_QUANTILES(lighthouse_pwa_score, 101)[
+  OFFSET
+    (51)], 2) AS p50,
+  ROUND(APPROX_QUANTILES(lighthouse_pwa_score, 101)[
+  OFFSET
+    (76)], 2) AS p75,
+  ROUND(APPROX_QUANTILES(lighthouse_pwa_score, 101)[
+  OFFSET
+    (91)], 2) AS p90
+FROM (
+  SELECT
```

Comment:
Can you rewrite this without the nested query? Eg https://github.com/HTTPArchive/bigquery/blob/master/sql/timeseries/responsiveImages.sql

You'll be duplicating the UDF call to getPWAScore but I think it's ok.

Reply:
Sounds good, changed.